cmd/integration/harness.go

```diff
@@ -78,20 +78,25 @@ func NewHarness(cfg *ServerConfig, args []string) (*Harness, error) {
 }
 
 func main() {
+	dummyChan := make(chan string)
 	log.JSONLog(true)
 	// os.Args[0] contains the current process path
 	h, err := NewHarnessDefaultServerConfig(os.Args[1:])
 	if err != nil {
 		log.With().Error("An error has occurred while generating a new harness: ", log.Err(err))
 	}
 	// listen on error channel, quit when process stops
-	for {
-		select {
-		case errMsg := <-h.server.errChan:
-			log.With().Error("harness received an err from subprocess: ", log.Err(errMsg))
-		case <-h.server.quit:
-			log.With().Info("harness got a quit signal from subprocess")
-			return
+	go func() {
+		for {
+			select {
+			case errMsg := <-h.server.errChan:
+				log.With().Error("harness received an err from subprocess: ", log.Err(errMsg))
+			case <-h.server.quit:
+				log.With().Info("harness got a quit signal from subprocess")
+				break
+			}
 		}
-	}
+	}()
+
```

Comment:
add a log here so that we'll know we got here

Reply:
done 805ce369aac4a6138fa880d24e01e0a5dbf9d99a